opentracing-mock/pom.xml

```diff
@@ -36,6 +36,14 @@
             <groupId>${project.groupId}</groupId>
             <artifactId>opentracing-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>io.opentracing</groupId>
+            <artifactId>opentracing-noop</artifactId>
+        </dependency>
+        <dependency>
+            <groupId>io.opentracing</groupId>
```

Comment:
btw, to follow up on this one point @pavolloffay: `ot-util` has tests that depend on MockTracer.